test/k8s-e2e/e2e-test-execute

```diff
@@ -219,23 +373,84 @@ function gather_test_summary_data_for_elastic_search() {
 		declare -a array
 		mapfile -t array <<<"$(echo "$junit_summary" | grep -oE "[0-9.]+")"
 		if [ ${array[0]} -eq 0 ]; then
-			echo "test with search string \"${e2e_test}\" was not found/executed"
+			warning "WARNING: test with search string \"${e2e_test}\" was not found/executed. e2e.log output:"
+			sed -n '1h;1!H;${;g;s/.*\(------------------------------.*------------------------------\).*/\1/p;}' "$e2e_log".tmp
 		fi
 		executed_testcases=$(echo "$executed_testcases + ${array[0]}" | bc)
 		failed_testcases=$(echo "$failed_testcases + ${array[1]}" | bc)
 		successful_testcases=$(echo "$successful_testcases + ${array[0]} - ${array[1]}" | bc)
 		test_duration=$(echo "$test_duration + ${array[2]}" | bc)
+		test_duration=${test_duration%.*}
 		if [ "$failed_testcases" -eq "0" ]; then
 			testsuite_successful="true"
 		else
 			testsuite_successful="false"
 		fi
 		echo "Test status: executed_testcases: $executed_testcases, successful_testcases: $successful_testcases, failed_tests: $failed_testcases, test_duration: $test_duration,  testsuite_successful: $testsuite_successful"
 	else
-		echo "Couldn't read junit file."
+		echo -e "$(warning WARNING:) Couldn't read junit file."
 	fi
 }
 
+
+function pushResutlsToGCS() {
+	# Push result to GCS (Google Cloud Storage)
+
+	local google_bucket_secrets_file="gardener-logs-conformance-tests.json"
+
+	# Download Google SDK
+	google_sdk="google-cloud-sdk-226.0.0-linux-x86_64.tar.gz"
```

Comment:
maybe we can add this to our base image later

Reply:
Would make sense, yes